concent_api/core/exceptions.py

```diff
@@ -1,11 +1,19 @@
+from utils.constants import ErrorCode
+
 
 class UnexpectedResponse(Exception):
     pass
 
 
-class Http400(Exception):
+class ConcentInSoftShutdownMode(Exception):
     pass
 
 
-class ConcentInSoftShutdownMode(Exception):
-    pass
+class Http400(Exception):
+
```

Comment:
this line could be removed

Reply:
From pep8:

> Method definitions inside a class are surrounded by a single blank line.

Reply:
I don't like these extra lines too, but if PEP8 says so then we have no choice :)

Reply:
Yeah... I kill them on sight :)